source/content/guides/decoupled/drupal-backend-starters/02-create.md

```diff
@@ -19,23 +19,27 @@ integration: [--]
 
 This section provides information on how to use the Drupal Backend Starter for Front-End Sites.
 
-## Choose an Installation Option
+## Choose a Creation Option
 
-You can install the backend starter kit with Build Tools or with the Dashboard Upstream. Review the information below to select the right option for your setup.
+You can create your backend starter kit project with the Dashboard Upstream, the Terminus Decoupled Kit Plugin, or with Build Tools. Review the information below to select the right option for your setup.
 
 ### Use the Dashboard Upstream if:
 
 - You require a simpler setup
 
 - The Pantheon repository is your source of truth
 
+### Use the Dashboard Upstream if:
```

Comment:
I'm assuming this isn't supposed to be 'Dashboard Upstream.' 'Use the Terminus Decoupled Kit Plugin,' perhaps? If that is the case, maybe the sub bullet should be 'You would like to create a related back-end and front-end codebase in a single process'